### DISCUSSION TITLE
OpenUI Key

### DISCUSSION THREAD

**Discussion by mypcisamiga** [sacredbanana/AmigaGPT#16] [Q&A] [answered]
I'm a novice at this.  I got a key from OpenUI but when I use it I get a out of quota error in ChatCPT.   Does AmigaGPT require a paid usage for OpenUI or is it possible to use it with a limited free option from OpenUI?

**Comment by sacredbanana** [comment, marked as answer]
You now have to purchase credit in your API account unfortunately. This is regardless if you have a ChatGPT Plus subscription or not

**Comment by sacredbanana** [comment]
you can add credits here

https://platform.openai.com/settings/organization/billing/overview
